modules/Forum/pages/forum/edit.php

```diff
@@ -120,6 +119,7 @@
         if ($validation->passed()) {
             // Valid post content
             $content = Output::getClean(Input::get('content'));
+            $content = MentionsParser::parse($user->data()->id, $content, URL::build('/forum/topic/' . $topic_id, 'pid=' . $post_id), ['path' => ROOT_PATH . '/modules/Forum/language', 'file' => 'forum', 'term' => 'user_tag'], ['path' => ROOT_PATH . '/modules/Forum/language', 'file' => 'forum', 'term' => 'user_tag_info', 'replace' => '{x}', 'replace_with' => Output::getClean($user->data()->nickname)]);
```

Comment:
_Maybe_ worth adding a parameter to `parse` to disable sending alerts when editing a post?